pymdownx/blocks/block.py

```diff
@@ -232,53 +248,55 @@ def __init__(self, length, tracker, block_mgr, config):
         self.length = length
         self.tracker = tracker
         self.md = block_mgr.md
-        self.arguments = []
-        self.options = {}
+        self.arguments: list[Any] = []
+        self.options: dict[str, Any] = {}
         self.config = config
         self.on_init()
 
-    def is_raw(self, tag):
+    def is_raw(self, tag: etree.Element) -> bool:
         """Is raw element."""
 
         return self._block_mgr.is_raw(tag)
 
-    def is_block(self, tag):  # pragma: no cover
+    def is_block(self, tag: etree.Element) -> bool:  # pragma: no cover
         """Is block element."""
 
         return self._block_mgr.is_block(tag)
 
-    def html_escape(self, text):
+    def html_escape(self, text: str) -> str:
         """Basic html escaping."""
 
         text = text.replace('&', '&amp;')
         text = text.replace('<', '&lt;')
         text = text.replace('>', '&gt;')
         return text
 
-    def dedent(self, text, length=None):
+    def dedent(self, text: str, length: int | None = None) -> str:
         """Dedent raw text."""
 
         if length is None:
             length = self.md.tab_length
 
-        min_length = float('inf')
+        min_length = length
```

Comment:
this is the only "logic change", if you can call it that, avoiding the float in the mix between integer comparisons, and relying on the commutativity of `min` to start with the last value to `min` with.

Reply:
Just use `sys.maxsize` instead. You are correct though, a float really shouldn't be used here.

Reply:
Done